Chore fix docs examples

Narretz opened the pull request:
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Bug fix for the docs-app


**What is the current behavior? (You can also link to an open issue here)**
Docs app included incorrect files for the embedded / plnkr examples

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

I would like to test that the plnkrs open with the reference to the correct files (code.angularjs.org), but I think this is out of scope for an e2e test / too difficult. We should probably test the plnkrOpener directive instead.

Narretz committed:
revert: chore(docs-app): load example files based on active deployment

This reverts commit d91a6bdbc66098ff7bbf8fa435ae5652fe6a795f.

The runnable examples cannot rely on the CDN for loading the common files,
because the CDN push does not happen at the same time as the docs are generated,
which means the doc examples are non-functional for an unforeseeable time.

Narretz committed:
chore(doc-gen): include source files from code.angularjs.org for production plnkr examples

petebacondarwin commented on docs/config/services/deployments/production.js:
```diff
@@ -7,15 +7,15 @@ var angularCodeUrl = '//code.angularjs.org/';
-// The currentVersion may not be available on the cdn (e.g. if built locally, or hasn't been pushed
-// yet). This will lead to a 404, but this is preferable to loading a version with which the example
-// might not work (possibly in subtle ways).
+// The "examplesCdnUrl" here applies to the examples when they are opened in plnkr.co.
```
Is `plnkr.co.` a typo? Should it stop at `.co.`?

Narretz replied:
It's a full stop. ;)

petebacondarwin commented on docs/config/services/deployments/production.js:
```diff
@@ -7,15 +7,15 @@ var angularCodeUrl = '//code.angularjs.org/';
+// The embedded examples instead always include the files from the *default* deployment,
+// to ensure that the source files are always available.
+// The plnkr examples must always use the code.angularjs.org source files.
+// We cannot rely on the CDN files here, because they are not deployed during the same time
```
How about

> because they are not deployed by the time the docs app needs them.

petebacondarwin commented on docs/config/services/deployments/production.js:
```diff
@@ -7,15 +7,15 @@ var angularCodeUrl = '//code.angularjs.org/';
+// The plnkr examples must always use the code.angularjs.org source files.
+// We cannot rely on the CDN files here, because they are not deployed during the same time
+// as the docs app.
 var examplesCdnUrl = versionInfo.currentVersion.isSnapshot ?
```
Since it is no longer an actual CDN let's change this variable to something like `examplesPath`.

petebacondarwin reviewed (approved):
Some minor changes but I think this works for me.

Narretz committed:
chore(docs.angularjs.org): fix docs deploy for source file changes

Since the embedded examples in the docs app now include the local source files instead
of CDN files, we need to include the source files into the folder that is deployed
to Firebase hosting
